devtools/readme.md

```diff
@@ -0,0 +1,199 @@
+<!---
+title='Devtools Day 1'
+time='120'
+facilitation=true
+[objectives]
+1="Inspect the DOM"
+2="Identify the values of the design tokens in the CSS"
+3="Alter the values of the CSS Variables using DevTools "
+4="Select the Grid Badge in the Elements panel"
+5="Use the Grid Editor to change layout"
+6="Run a Lighthouse audit"
+7="Identify and resolve the reported issues"
+8="Inspect the Accessibility Tree and compare it to the DOM"
++++
+-->
+
+# Inspector Gadget
+
+### Prep
+
+For this workshop you will need:
+
+- [ ] [Chrome](https://www.google.com/intl/en_uk/chrome/) with Devtools
+- [ ] A [ChatGPT account](https://chat.openai.com/auth/login)
+- [ ] This [Devtools Workshop page](http://cyf-workshops.netlify.app/devtools/)
+- [ ] Someone to keep track of the timer for each activity
+
+You've already used a little bit of Devtools in ITD when you used [Lighthouse](https://developer.chrome.com/docs/lighthouse/) to audit your code. But there's much much more! Devtools is an [entire IDE](https://developer.chrome.com/) running inside your browser and it has hundreds of ways to inspect, test, and develop your code.
+
+<!--
+{{<note type="tip" title="Explore">}} -->
+
+Each browser has its own Devtools. Download Safari, Chrome Canary, and Firefox Developer Edition to explore the differences.
+
+<!-- {{</note>}} -->
+
+## Inspect the DOM
+
+```objectives
+- [ ] Inspect the DOM
+- [ ] Identify the values of the design tokens in the CSS
+- [ ] Alter the values of the CSS Variables using DevTools
+```
+
+<!--
+{{<note type="exercise" title="Explore">}} -->
+
+In groups of no more than five.
+
+1.  Set a [whole class timer for 10 minutes](https://www.google.com/search?q=timer+for+10+minutes).
+1.  [Inspect the page](http://cyf-workshops.netlify.app/devtools/) with Devtools.
+1.  _Write down_ the answers to these questions:
+
+- What is the name of the font used on the page?
+- What is the CSS variable that holds this value?
+- What is the brand colour value?
+- What variables hold [CSS length data type](https://developer.mozilla.org/en-US/docs/Web/CSS/length) values?
+
+1. When the timer goes off, share your answers with the other groups. Paste them in a thread in Slack and go quickly to the next challenge.
+<!-- {{</note>}} -->
+
+Fonts, colors, and spacing are often stored as CSS Variables, or custom properties. This allows them to be easily changed throughout a project by simply altering one value. Let's experiment with this.
+
+<!--
+{{<note type="exercise" title="Experiment">}} -->
+
+1. Set a [timer for 10 minutes](https://www.google.com/search?q=timer+for+10+minutes).
+2. Inspect the [CSS of the page](http://cyf-workshops.netlify.app/devtools/) with Devtools.
```

Comment:
I think 2. and 3. here should be the first tasks that trainees do at the start of the this group of tasks ( just because the subsequent tasks make reference to CSS variables )

Reply:
I disagree because the first task is just locating the variables in the panel. It will take many people 10 minutes simply to do that.

The second task is actually exploring and modifying things, which is a more complicated task.